src/test/ui/parser/increment.stderr

```diff
@@ -0,0 +1,52 @@
+error: expected expression, found `+`
+  --> $DIR/increment.rs:3:7
+   |
+LL |     i++;
+   |       ^ expected expression
+   |
+   = note: Rust has no dedicated increment operator
+help: try using `+= 1` instead
+   |
+LL |     i += 1;
+   |       ~~~~
+
+error: expected expression, found `+`
+  --> $DIR/increment.rs:8:13
+   |
+LL |     while i++ < 5 {
+   |             ^ expected expression
+   |
+   = note: Rust has no dedicated increment operator
+help: try using `+= 1` instead
+   |
+LL |     while i += 1 < 5 {
```

Comment:
I'm not sure how to fix this suggestion so it works like the pre-increment suggestion. The issue is that by the time we get to the second `+`, we've forgotten about the `i` token, so I don't see a good way of getting the starting span.

Reply:
Note to self: If we fix this, I should make sure to add `// run-rustfix` to the test.

Reply:
Fixed in newer version (still working on it, so haven't pushed).